src/useChat.tsx

```diff
@@ -189,7 +189,7 @@ export const useChat = (initValues?: ChatMessage[], initModel?: string) => {
 
   const onUpdateConversationTitle = useCallback(
     async (id: string, newTitle: string) => {
-      await updateConversation(id, { title: newTitle, lastSaved: Date.now() });
```

Comment:
Note: adjusting a title shouldn't "save it as new" bringing it to the top of the history, that's reserved for when a new message is added only.